src/Components/NavBar/NavItem.tsx

```diff
@@ -73,7 +73,7 @@ export const NavItem: React.FC<NavItemProps> = ({
     if (isString(navItemLabel) || label)
       trackEvent({
         action_type: AnalyticsSchema.ActionType.Hover,
-        subject: navItemLabel.toString() || label,
+        subject: label || navItemLabel.toString(),
```

Comment:
@damassi I remember why I initially checked for the label first in a previous PR. For the Artworks & Artists drop downs the child of `NavItem` is not a string. We pass `label` in as a prop in order to check the hover of the label.

Reply:
Yah I noticed the "label" prop last night, that actually clears a lot of little things up versus having to check `parentNode` and so on. It might be good to dig through navbar code and see where we can leverage label instead.